Larastan in action

@nunomaduro May I contribute a Travis test that runs Larastan on the latest laravel/laravel package?

Ignoring one error brings up two others in vanilla Laravel: `composer create-project --quiet --prefer-dist "laravel/laravel" laravel`

```
$ php artisan code:analyse --level=max
 19/19 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ --------------------------------------------------------------------------------------------
  Line   app/Exceptions/Handler.php
 ------ --------------------------------------------------------------------------------------------
  49     Method App\Exceptions\Handler::render() should return Illuminate\Http\Response but returns
         Symfony\Component\HttpFoundation\Response.
 ------ --------------------------------------------------------------------------------------------


 [ERROR] Found 1 error
```

and with

```neon
parameters:
    ignoreErrors:
        - '|^Method App\\Exceptions\\Handler::render\(\) should return Illuminate\\Http\\Response but returns Symfony\\Component\\HttpFoundation\\Response\.$|'
```

```
$ php artisan code:analyse --level=max -c /home/viktor/tmp/phpstan-laravel/laravel/phpstan.neon
 19/19 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ --------------------------------------------------------
  Line   app/Http/Controllers/Auth/RegisterController.php
 ------ --------------------------------------------------------
  66     Call to an undefined static method App\User::create().
 ------ --------------------------------------------------------

 ------ ------------------------------------------------------------------------------------
  Line   app/Providers/BroadcastServiceProvider.php
 ------ ------------------------------------------------------------------------------------
  17     Call to an undefined static method Illuminate\Support\Facades\Broadcast::routes().
 ------ ------------------------------------------------------------------------------------


 [ERROR] Found 2 errors
```

@nunomaduro Please advise.

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,8 +17,15 @@ env:
     - COMPOSER_FLAGS=""
 
 before_script:
-    - travis_retry composer self-update
-    - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-source
+  - travis_retry composer self-update
+  - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-source
 
 script:
-    - vendor/bin/phpunit --coverage-text --coverage-clover=coverage.xml
+  - vendor/bin/phpunit --coverage-text --coverage-clover=coverage.xml
+  - composer create-project --quiet --prefer-dist "laravel/laravel" laravel && cd laravel/
+  # Add package from source
+  - sed -e 's|"type": "project",|&\n"repositories": [ { "type": "vcs", "url": "../" } ],|' -i composer.json
+  - composer require --dev nunomaduro/larastan
+  # Fix https://github.com/laravel/framework/pull/23825
+  - sed -e 's|@return \\Illuminate\\Http\\Response$|@return \\Symfony\\Component\\HttpFoundation\\Response|' -i app/Exceptions/Handler.php
+  - php artisan code:analyse --level=max

